integration/firestore/karma.conf.js

```diff
@@ -14,57 +14,22 @@
  * limitations under the License.
  */
 
-const webpackConfig = require('./webpack.config');
+const karma = require('karma');
+const path = require('path');
+const karmaBase = require('../../config/karma.base');
 
-// Karma configuration
 module.exports = function(config) {
-  config.set({
+  const karmaConfig = Object.assign({}, karmaBase, {
+    // files to load into karma
+    files: [
+      `${path.dirname(require.resolve('firebase'))}/firebase.js`,
+      `${path.dirname(require.resolve('firebase'))}/firebase-firestore.js`,
```

Comment:
I commented out these lines for firebase.js and firebase-firestore.js and the tests still pass.

Reply:
Should be fixed now.